iconservice/iconscore/icon_score_class_loader.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 ICON Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib.util
+import json
+import os
+import sys
+
+from ..deploy.utils import get_package_name_by_address_and_tx_hash
+from ..deploy.utils import get_score_deploy_path
+from ..base.address import Address
+
+
+class IconScoreClassLoader(object):
+    """IconScoreBase subclass Loader
+
+    """
+
+    _PACKAGE_JSON_FILE = 'package.json'
+    _MAIN_SCORE = 'main_score'
+    _MAIN_FILE = 'main_file'
+
+    @staticmethod
+    def init(score_root_path: str):
+        if score_root_path not in sys.path:
+            sys.path.append(score_root_path)
+
+    @staticmethod
+    def exit(score_root_path: str):
+        sys.path.remove(score_root_path)
+
+    @staticmethod
+    def _load_package_json(score_deploy_path: str) -> dict:
+        """Loads package.json in SCORE
+
+        :param score_deploy_path:
+        :return:
+        """
+        pkg_json_path = os.path.join(score_deploy_path, IconScoreClassLoader._PACKAGE_JSON_FILE)
+        with open(pkg_json_path, 'r') as f:
+            return json.load(f)
+
+    @staticmethod
+    def run(score_address: 'Address', tx_hash: bytes, score_root_path: str) -> type:
```

Comment:
replace score_address, tx_hash to path

Reply:
Keep this code unchanged. More detail review is needed.